src/screens/ProfileScreen/elements/ProfilePicture.js

```diff
@@ -51,7 +51,7 @@ const ProfilePicture = ({
                   style={styles.profileImage(size, width)}
                   source={{
                     uri: profilePicPath ? `${profilePicPath}` : DEFAULT_PROFILE_PIC_PATH,
-                    cache: 'cacheOnly'
+                    cache: 'web'
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

Changing the cache policy from `'cacheOnly'` to `'web'` may increase the load time of the profile picture as it will always fetch the image from the web. If the profile picture doesn't change frequently, consider using a different caching strategy like `'immutable'`, which will keep the image in the cache until it is removed.
```diff
- cache: 'web'
+ cache: 'immutable'
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->